### ISSUE TITLE
Add more skills

### ISSUE THREAD

**Issue by MayankB11** [krssg-ssl/skills_py#5]
## Skills to add : 
- [x]  sDefendPoint
- [ ]  sDribbleToPoint
- [x]  sDribbleTurn
- [ ]  sGoalKeeping
- [ ]  sGoToBall
- [x]  sGoToPoint
- [x]  sKickToGoal
- [x]  sKickToPoint
- [ ]  sReceiveBall
- [x]  sTurnToAngle
- [x]  sTurnToPoint

If there is an error in any skill, discuss and correct it.
Comment here whichever skill you are working on so that two persons don't work on the same  

Deadline : 16th May, 2017.
@americast @ashishkg0022 @sipah00

**Comment by americast** [comment]
@MayankB11 I would like to work on sTurnToAngle.

**Comment by ashishkg0022** [comment]
i will work on sDribbleTurn ,  sKickToGoal , sKickToPoint  , sDefendPoint for now .

**Comment by sipah00** [comment]
I will work on sGoToBall, sGoToPoint,sTurnToPoint for now.

**Comment by MayankB11** [comment]
@americast Instead of  "return getRobotCommandMessage(botID, 0, 0, omega, 0, true)" it should use the send_command method defined in the skill_node module. Take reference from the tactics that were already completed like sKick.py

**Comment by ashishkg0022** [comment]
@MayankB11 sKickToGoal has no class in skills_union.py . so how to send the power value in the send_command method ?

**Comment by annimesh2809** [comment]
> @MayankB11 sKickToGoal has no class in skills_union.py . so how to send the power value in the send_command method ?

@ashishkg0022 I don't think we require sKickToGoal as sKickToPoint is more general and covers it.

**Comment by MayankB11** [comment]
Exactly, sKickToGoal just kicks to the centre of the goal with full power. So, all its parameters are pre-defined and its param can be None.

**Comment by sipah00** [comment]
@MayankB11 , in [sReceiveBall.cpp](https://github.com/krssg-ssl/skills/blob/master/src/sReceiveBall.cpp) ,at last line ---` return getRobotCommandMessage(botID, 0, 0, 0, 0, false); ` here passed parameters are temp(as mentioned for Testing) ...So I want to know actual parameters that should be passed here.

**Comment by MayankB11** [comment]
@sipah00  Since the ballPos is already in state, so param passed will be None. You can use the state to predict the ball position and adjust the bot's position and angle accordingly.

**Comment by americast** [comment]
@MayankB11 I would like to work on sReceiveBall, sGoalKeeping and sDribbleToPoint.

**Comment by MayankB11** [comment]
@ashishkg0022  Check for the errors in sKickToPoint.py
